tools/cwtool/CwTool.hs

```diff
@@ -91,6 +92,11 @@ topLevelCommands =
       "header-dump"
       "Dump Block Headers to a JSON array"
       HeaderDump.main
+  , CommandSpec
+      "decode" -- TODO: there is probably a better title to use here
```

Comment:
Since the variants are nested beneath this top-level command, maybe use `b64`?  And below possibly `Command line utils for Chainweb's base64 encode/decode`?